### PR TITLE
Added in Google Analytics

### PR DESCRIPTION
**Pull request by ghost**
The Google Analytics script has been included just before the end </head> tag to capture statistics on who is visiting the website.
#1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,13 +6,23 @@
...
+    ga('send', 'pageview');
+  </script>
 </head>
 <body>
     <h1>The Black Binder</h1>
     <h2>Your number one directory for Black Culture</h2>
     <p>Enter your email address and we will let you know when we are almost ready to go live</p>
-    <input type="text" name="Email" value="Your email address...">
-    <button type="button">Notify Me</button>
+    <form method="POST" action="http://formspree.io/hello@theblackbinder.com">
```
Comment (tanyapowell) [inline index.html]: 😄 you didn't create this branch from `master`, did ya?

File: index.html [hunk 2]
```diff
@@ -6,13 +6,23 @@
   <link rel="stylesheet" href="./style.css">
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.4/css/bootstrap.min.css" integrity="sha384-2hfp1SzUoho7/TsGGGDaFdsuuDL0LX2hnUp6VkX3CUQ2K4K+xjboZdsXyp4oUHZj" crossorigin="anonymous">
   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" integrity="sha384-T8Gy5hrqNKT+hzMclPo118YTQO6cYprQmhrYwIiQ/3axmI1hQomh7Ud2hPOy8SP1" crossorigin="anonymous">
+  <script>
```
Comment (tanyapowell) [inline index.html]: To help with performance can this please be moved to under the `</body>` tag

It's better to load this script after the page has loaded

File: index.html (hunk 1 above)
Comment (tanyapowell) [reply index.html]: Please remove all this `form` stuff. They do not belong in this pull request

**Review by tanyapowell — CHANGES_REQUESTED** [review]

File: index.html [hunk 3]
```diff
@@ -11,12 +11,18 @@
     <h1>The Black Binder</h1>
     <h2>Your number one directory for Black Culture</h2>
     <p>Enter your email address and we will let you know when we are almost ready to go live</p>
-    <input type="text" name="Email" value="Your email address...">
-    <button type="button">Notify Me</button>
```
Comment (tanyapowell) [inline index.html]: Can you put this back in?

`<input type="text" name="Email" value="Your email address...">`
`<button type="button">Notify Me</button>`

**Review by tanyapowell — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (ghost) [reply index.html]: done

**Review by tanyapowell — APPROVED** [review]
👍

**Pull request merged**